examples-gallery/plot_betts_10_1.py

```diff
@@ -0,0 +1,165 @@
+"""
+Alp Rider
+=========
+
+This is example 10.1 from Betts' book "Practical Methods for Optimal Control
+Using NonlinearProgramming", 3rd edition, Chapter 10: Test Problems.
+More details are in sectionn 4.11.7 of the book.
+
+The main issue here is an inequality constraint for the state variables:
+:math:`y_1^2 + y_2^2 + y_3^2 + y_4^2 \\geq function(time, parameters)`
+
+As presently *opty* does not support inequality constraints, I introduced a new
+state variable `fact` to take care of the inequality. The inequality is then
```

Comment:
```suggestion
state variable ``fact`` to take care of the inequality. The inequality is then
```

double back ticks give monspace font